api-mock/api-mock-server.py

```diff
@@ -0,0 +1,156 @@
+import json
```

Comment:
mock server should go in python sdk

Reply:
yes, but I'm waiting for the final version to move it